user-guide/Advanced_Functionality/Databases/Indexing_database/OpenSearch_Database/Installing_OpenSearch_database.md

```diff
@@ -50,7 +50,7 @@ These are the main steps of the setup:
   # Use a descriptive name for your cluster:
   #
   cluster.name: NameOfYourCluster
-  cluster.initial_master_nodes: ["opensearchnode1"]
+  cluster.initial_manager_nodes: ["opensearchnode1"] 
```

Comment:
Nuance: we support both Opensearch 1 and 2. 
2 had "initial_manager_nodes" as per https://github.com/opensearch-project/OpenSearch/commit/19eadb46ba80df1750402a3faad55114823087f2

1 has "initial_master_nodes". As such we should document both.

Reply:
Added clarification to the initial master/manager node setting to use for OpenSearch 1 and 2